deploy/cr-minimal.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: psmdb.percona.com/v1-9-0
+kind: PerconaServerMongoDB
+metadata:
+  name: minimal-cluster
+#  finalizers:
+#    - delete-psmdb-pvc
+spec:
+  crVersion: 1.9.0
+  image: percona/percona-server-mongodb:4.4.6-8
+  allowUnsafeConfigurations: true
+  updateStrategy: SmartUpdate
+  upgradeOptions:
+    versionServiceEndpoint: https://check.percona.com
+    apply: 4.4-recommended
+    schedule: "0 2 * * *"
+  secrets:
+    users: minimal-cluster-secrets
```

can we remove `image`, `updateStrategy`, `versionServiceEndpoint`, `secrets.users`?
```suggestion
  crVersion: 1.9.0
  allowUnsafeConfigurations: true
  upgradeOptions:
    apply: 4.4-recommended
    schedule: "0 2 * * *"
```

---

Uploaded new version. It is as minimal as it gets. 
Removing secrets is possible, but then the weirdly named secret is created: percona-server-mongodb-users

lets keep it to be explicit and ease the docs.